keps/sig-apps/961-maxunavailable-for-statefulset/README.md

```diff
@@ -229,29 +420,33 @@ https://github.com/kubernetes/kubernetes/blob/v1.13.0/pkg/controller/statefulset
 ...
 ```
 
-### Risks and Mitigations
-We are proposing a new field called `maxUnavailable` whose default value will be 1. In this mode, StatefulSet will behave exactly like its current behavior.
-Its possible we introduce a bug in the implementation. The mitigation currently is that is disabled by default in Alpha phase for people to try out and give
-feedback. 
-In Beta phase when its enabled by default, people will only see issues or bugs when `maxUnavailable` is set to something greater than 1. Since people have
-tried this feature in Alpha, we would have time to fix issues.
+### Test Plan
 
+<!--
+**Note:** *Not required until targeted at a release.*
+The goal is to ensure that we don't accept enhancements with inadequate testing.
 
-### Upgrades/Downgrades
+All code is expected to have adequate tests (eventually with coverage
+expectations). Please adhere to the [Kubernetes testing guidelines][testing-guidelines]
+when drafting this test plan.
 
-We will default to 1 for maxUnavailable field in StatefulSet for backward compatibility
+[testing-guidelines]: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
+-->
 
-Downgrades
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
 
-When downgrading from a release with this feature, to a release without maxUnavailable, there are two cases
- - If maxUnavailable is greater than 1, there are two more cases:-
-   - If you're rolling back to a release that doesn't have this field - then there is even no way to discover it
-   - If you're just disabling the feature (either together with downgrade to a release that has a field or without downgrade),the field should remain set 
-        (unless someone will explicitly delete it later), but controller should ignore its behavior (and there shouldn't be a way to set it if the feature gate
-         is switched off).
- - If maxUnavailable is less than equal to 1 -- in this case user wont see any difference in behavior
+##### Prerequisite testing updates
+
+<!--
+Based on reviewers feedback describe what additional tests need to be added prior
+implementing this enhancement to ensure the enhancements have also solid foundations.
+-->
 
-### Tests
+No.
```

Comment:
This no doesn't mean anything, this section should outline any  additional tests that are needed, since we don't have any, yet, just leave it empty, for now.

Reply:
At the same time you're missing 3 important section that are required:
- unit tests
- integration tests
- e2e tests
see the template: https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md#test-plan